adapter/internal/discovery/xds/server.go

```diff
@@ -701,9 +697,7 @@ func DeleteAPIWithAPIMEvent(uuid, organizationID string, environments []string,
 			for _, environment := range environments {
 				// delete environment if exists
 				delete(apiUUIDToGatewayToVhosts[uuid], environment)
-				if gaEnabled && revisionUUID != "" {
```

Comment:
Don't we have to move `gaEnabled` check into the method?

Reply:
no, ga is not needed to be enabled to send the revision ack.